weave/Scripts/Player.cs

```diff
@@ -12,7 +12,7 @@ public partial class Player : CharacterBody2D
     [Signal]
     public delegate void PlayerShotBulletEventHandler(Node2D bullet, Vector2 globalPosition);
 
-    private const int MovementSpeed = 100;
+    private float MovementSpeed = 100f;
```

Comment:
_movementSpeed

Reply:
I think you will get syntax highlighting for this if you have the sonarlint extension